src/navigation/TabNavigator.tsx

```diff
@@ -68,6 +69,14 @@ export default function TabNavigator() {
           tabBarIcon: ({ color }) => <Icon name="plus" color={color} />,
         }}
       />
+      <Screen
+        name="Search"
```

Comment:
We created a new tab for Search but we should probably restructure Navigation at some point @mylesdomingo

Reply:
it depends on the designs I need finalized navigation flows @kennethlien

Reply:
Yes, @kbrgl is working on those